2996_reflection/reflection.md

```diff
@@ -5423,17 +5427,41 @@ and define an expression `$INVOKE-EXPR$` as follows:
 ::: std
 ::: addu
 
+```cpp
+template <class T> requires constructible_from<u8string, T>
+consteval data_member_options_t::name_type(T&& value);
+
+template<class T> requires constructible_from<string, T>
+consteval data_member_options_t::name_type(T&& value);
+```
+[1]{.pnum} *Postconditions*:
+
+- [#.#]{.pnum} `$get-u8string$()` is equal to `u8string(value)`, or `$get-string$()` is equal to `string(value)`, respectively.
+- [#.#]{.pnum} `$is-utf8$()` is `true` or `false`, respectively.
```

Comment:
If we word this as a variant, then just add wording to each constructor that it constructs the variant in the appropriate way.

Reply:
Done - Let me know if there's a nicer way to word it.